FabricExample/ios/ReactNativeDelegate.swift

```diff
@@ -0,0 +1,16 @@
+import React
+import React_RCTAppDelegate
+
+class ReactNativeDelegate: RCTDefaultReactNativeFactoryDelegate {
+  override func sourceURL(for bridge: RCTBridge) -> URL? {
+    self.bundleURL()
+  }
+
+  override func bundleURL() -> URL? {
+    #if DEBUG
+      RCTBundleURLProvider.sharedSettings().jsBundleURL(forBundleRoot: "index")
+    #else
+      Bundle.main.url(forResource: "main", withExtension: "jsbundle")
+    #endif
+  }
+}
```

Comment:
Is this delegate needed? does it do something more than `RCTDefaultReactNativeFactoryDelegate`?

Reply:
Makes sense now. Thanks